cmd/test/main_test.go

```diff
@@ -278,6 +280,18 @@ func test_testCommand_only_k8s_files(t *testing.T, k8sValidator *K8sValidatorMoc
 	k8sValidator.AssertCalled(t, "GetK8sFiles", mock.Anything, 100)
 }
 
+
+func test_testCommand_demo_yaml(t *testing.T, evaluator *mockEvaluator, k8sValidator *K8sValidatorMock, filesConfigurations []*extractor.FileConfigurations, evaluationId int, ctx *TestCommandContext) {
+	test(ctx, []string{"8/*"}, TestCommandFlags{OnlyK8sFiles: true, Output: "yaml"})
```

Comment:
Hello @Kanika637 ,  add an `&` before `TestCommandFlags`. `test` need a pointer-address instead of literal address.

Reply:
@Abhra303  I have committed the changes.

Reply:
Sorry @Kanika637 I didn't look at it then - you have to use `TestCommandOptions` instead of `TestCommandFlags`

Reply:
Okk @Abhra303  Let me change it to that then.

Reply:
Done with the changes

Reply:
Yes actually I was figuring out why these linting issues are coming again and I think it's somewhat because of the changes made in main.go file. It would be very helpful if you could guide me through it.